Sources/ApolloCodegenLib/ApolloCodegen.swift

```diff
@@ -76,14 +79,14 @@ public class ApolloCodegen {
       rootURL: rootURL
     )
 
-    try validate(config: configContext)
+    try validate(context: configContext)
```

Comment:
How about this? - https://github.com/apollographql/apollo-ios/pull/2760/commits/a492d548b002f33c62af0ec2ab30633e45b5d0fb
* Takes the suggestion of the nameless parameter
* Keeps `compilationResult` as a second param but using the label `with` because it's still validation of the configuration but dependent on the compilation results rather than validating the compilation results.